docs/docs/install/environment-variables.md

```diff
@@ -51,6 +51,7 @@ Regardless of filesystem, it is not recommended to use a network share for your
 | `CPU_CORES`                         | Amount of cores available to the immich server  | auto-detected cpu core count | server                   |                    |
 | `IMMICH_API_METRICS_PORT`           | Port for the OTEL metrics                       |            `8081`            | server                   | api                |
 | `IMMICH_MICROSERVICES_METRICS_PORT` | Port for the OTEL metrics                       |            `8082`            | server                   | microservices      |
+| `IMMICH_BUILD_LOCK_PATH`            | Path of build-lock.json                         |     `./build-lock.json`      | server                   | api                |
```

Comment:
I don't think we documented the other build metadata env vars right?

Reply:
(As in, I don't think we need to document this one either)

Reply:
What do you mean by 'build metadata env'? We documented `IMMICH_WEB_ROOT` and `IMMICH_REVERSE_GEOCODING_ROOT`

Reply:
I don't think we need to or should document build env related stuff on this page, which should just be user facing env settings.